hadoop-hdds/annotations/src/main/java/org/apache/ozone/annotations/ReplicateAnnotationProcessor.java

```diff
@@ -67,14 +85,14 @@ private void checkMethodSignature(Element element) {
       return;
     }
     final ExecutableElement executableElement = (ExecutableElement) element;
-    final List<? extends TypeMirror> exceptions =
-        executableElement.getThrownTypes();
+    final boolean found = executableElement.getThrownTypes().stream()
+        .map(TypeMirror::toString)
```

Comment:
`toString` may depend on the Java vendor.  It seems the correct way is to use `processingEnv.getTypeUtils()` as mentioned in the javadoc https://docs.oracle.com/javase/8/docs/api/javax/lang/model/type/TypeMirror.html
> Types should be compared using the utility methods in [Types](https://docs.oracle.com/javase/8/docs/api/javax/lang/model/util/Types.html). ...

Reply:
Thanks for the pointer.  I'll make the change, although using `toString` is not new to the patch.